chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/SurveillanceController.java

```diff
@@ -668,4 +680,19 @@ private void checkUploadedSurveillanceOwnership(Surveillance pendingSurv) {
 			} 
 		}
 	}
+	
+	private void updateAuthority(Surveillance surv){
```

Comment:
The ONC_STAFF issue was discussed in the comment above this one. This is my reply:

> I think I misinterpreted the requirement in OCD-1180 for "ONC Admin" and OCD-1250 for "ONC user" as being ROLE_ONC_STAFF. I think I need to change references to ROLE_ONC_STAFF to ROLE_ADMIN.
>